src/locales/index.js

```diff
@@ -18,20 +18,70 @@ const mobileTranslations = {
   es: esMobile
 }
 
-const unsupportedLocaleTag = () => {
+const defaultLanguage = () => {
+  const polyglot = new Polyglot({locale: defaultLocaleTag, phrases: availableTransations[defaultLocaleTag] , onMissingKey: () => null})
+  if (!isDesktop) polyglot.extend(mobileTranslations[defaultLocaleTag])
```

Comment:
you might be able to reuse extendPolyglot, are any of the init parameters mandatory?

Reply:
Yes, I can reuse extendPolyglot actually. No, parameters are not mandatory, the instance can be created and extended later